#src/scss/blocks/header.scss

```
@@ -1,3 +1,12 @@
-h1 {
-    color: orange;
-}
\ No newline at end of file
+.header__row {
+    margin-left: -51px;
```

what is -51px? will it look the same on all screens, if it is just fixed pixels?

---

You could do:
```suggestion
    margin-left: -50px;
    margin-right: -50px;
```
I don't see any reasong giving auto to margin-right